examples/logging/tutorial-server-prometheus/README.md

```diff
@@ -0,0 +1,157 @@
+# DeepSparse Server + Prometheus/Grafana
```

Comment:
This is a duplicate of the other tutorial README.md right? This one should be way more complex (prometheus operator setup etc.) :)

@rsnm2 I propose the following.
This tutorial is a bit of a big boi. Also, Mark expressed his interest in having here some more details on how the users can pull the system metrics directly from the operator.
Let's not try to bite off too large of a piece. I propose to hold off with this tutorial for now and focus on the documentation. Once this is in a good shape, we can come back to tutorials, freshen them up, put some lipstick and test them. I am just not sure about the deadlines, we can sync on that.

Reply:
Yes, this is an updated version of the one you originally made. We should hold off on making the more complex one